### DISCUSSION TITLE
Source of business terms from the 2022 amendment

### DISCUSSION THREAD

**Discussion by jpmckinney** [OP-TED/eForms-SDK#197] [Q&A] [answered]
I know this is not the repository's responsibility, but where can I find the latest annex to the regulation? The annex linked from https://single-market-economy.ec.europa.eu/single-market/public-procurement/digital-procurement/eforms_en is old and doesn't include new BTs like BT-717.

**Comment by YvesJo** [comment]
Hi,
The 2022 amendment to the regulation isn't published yet and I recommend to preferably address such questions to the TED helpdesk who may follow up with DG GROW to get an estimation of when publication would happen.
The new BTs are already part of the SDK fields and will be integrated in the other SDK artefacts in the coming weeks.
KR.

**Comment by jpmckinney** [comment, marked as answer]
The answer is to download the file at https://ec.europa.eu/info/law/better-regulation/have-your-say/initiatives/13550-Public-procurement-eForms-amending-the-Implementing-Regulation-on-standard-forms-used-for-publication-of-notices_en

**Comment by ferrakaop** [comment]
To complete this business question, the 2022 amendment to the eForms regulation was published on 25 November 2022 at https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3A32022R2303

The final regulation amendment has the same new BTs as the annex published in the August-September public consultation but some of the texts (BT names and descriptions) were revised or clarified.

The "extended annex" Excel sheet has also been updated and was published just before Christmas at https://ec.europa.eu/docsroom/documents/52697 

https://single-market-economy.ec.europa.eu/single-market/public-procurement/digital-procurement/eforms_en has been updated to link to these 2022 sources.

**Reply by jpmckinney** [reply]
Thank you for the update!
